### PR TITLE
[SMBC] new Table Creation Mode: Empty Rows

### PR DESCRIPTION
**Pull request by sepp4me**
new Table Creation Mode: Empty Rows

### CODE REVIEW

**Comment by albertmink** [comment]
Thanks for the inclusive language 💙

File: file-formats/smbc/type/zif_aff_smbc_v1.intf.abap [hunk 1]
```diff
@@ -46,11 +46,14 @@ INTERFACE zif_aff_smbc_v1
...
-      "! By clicking on 'Create' a new line is created but automatic navigation to subpage is not triggered.
+      "! By clicking on 'Create' a new line is created but automatic navigation to subpage is not performed.
       inline       TYPE string VALUE 'Inline',
       "! <p class="shorttext synchronized">Creation Row</p>
-      "! By clicking on 'Add row' a new line is created but automatic navigation to subpage is not triggered.
+      "! By clicking on 'Add row' a new line is created but automatic navigation to subpage is not performed.
       creation_row TYPE string VALUE 'CreationRow',
+      "! <p class="shorttext synchronized">Empty Rows</p>
+      "! In create or edit mode, two new empty rows are added to the table.
+      inline_creation_rows TYPE string VALUE 'InlineCreationRows',
```
Comment (schneidermic0) [inline file-formats/smbc/type/zif_aff_smbc_v1.intf.abap:56]: Any reason why the short text "Empty Rows" an the constant name and value (inline creation rows) differ in their naming? Can we align this?

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
@sepp4me Thank you for your update. 👍 

See my question below

File: file-formats/smbc/type/zif_aff_smbc_v1.intf.abap (hunk 1 above)
Comment (sepp4me) [reply file-formats/smbc/type/zif_aff_smbc_v1.intf.abap]: The corresponding UI5 functionality is described here (section Odata V4): https://sapui5.hana.ondemand.com/sdk/#/topic/cfb04f0c58e7409992feb4c91aa9410b.html
empty_row

Comment (schneidermic0) [reply file-formats/smbc/type/zif_aff_smbc_v1.intf.abap]: OK, I see. :/

**Review by schneidermic0 — APPROVED** [review]
Looks good to me

**Pull request merged**